templates/test-sw.hbs

```diff
@@ -44,22 +44,22 @@
 
     <script>
       mocha.setup({
-        ui: 'bdd'
+        ui: 'bdd',
+        timeout: 10000,
       });
 
       chai.should();
-      const expect = chai.expect;
+      window.expect = chai.expect;
     </script>
 
     <!-- Package-specific tests matching packages/<pkg>/test/browser/*.js -->
 
     <script>
     describe('{{pkg}} > test/sw/*.js', function() {
-      {{#each scripts}}
-      it(`should run '{{this}}' sw tests`, function() {
```

Comment:
I think that's fine, but then those tests would have to live in a separate directory, or have an identifiably different naming structure or something. Otherwise they'd be included in the test bundle.

Since we don't (AFAICT) have any instances of this currently, do you want to wait until the use case comes up?